PyInstaller/building/osx.py

```diff
@@ -53,7 +53,12 @@ def __init__(self, *args, **kws):
             # Fallback to appname.
             self.bundle_identifier = self.appname
 
-        self.info_plist = kws.get('info_plist', None)
+        self.info_plist = kws.get('info_plist', [])
```

Comment:
If `info_plist` is not given as an argument, `self.info_plist` will become a list, and the next line will fail:
```
>>> [].get('a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'list' object has no attribute 'get'
```

Reply:
Yeah, stupid mistake. 👍